test/e2e/fixtures/crane_empty_img_tmpl/config.json

```diff
@@ -0,0 +1,65 @@
+{
+  "architecture": "amd64",
+  "config": {
+    "Hostname": "",
+    "Domainname": "",
+    "User": "",
+    "AttachStdin": false,
+    "AttachStdout": false,
+    "AttachStderr": false,
+    "Tty": false,
+    "OpenStdin": false,
+    "StdinOnce": false,
+    "Env": [
+      "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+      "foo="
+    ],
+    "Cmd": null,
+    "Image": "",
+    "Volumes": null,
+    "WorkingDir": "",
+    "Entrypoint": null,
+    "OnBuild": null,
+    "Labels": null
+  },
+  "container": "622070b2ab6961b448c41a0ca3f694e7301ceaef4676aafc0f5302500a17300d",
+  "container_config": {
+    "Hostname": "622070b2ab69",
+    "Domainname": "",
+    "User": "",
+    "AttachStdin": false,
+    "AttachStdout": false,
+    "AttachStderr": false,
+    "Tty": false,
+    "OpenStdin": false,
+    "StdinOnce": false,
+    "Env": [
+      "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+    ],
+    "Cmd": [
+      "/bin/sh",
+      "-c",
+      "#(nop) ",
+      "ENV foo="
+    ],
+    "Image": "",
+    "Volumes": null,
+    "WorkingDir": "",
+    "Entrypoint": null,
+    "OnBuild": null,
+    "Labels": null
```

Comment:
I think it would be smart to include our supported timestamp labels here, this would ensure our whitelist for this works (and make it easier to confirm #2699, but this is self-interest).

Reply:
I _think_ I just added what meant, although I am not sure about the value of adding labels with the same time value.

Reply:
You did, I will make some adjustments while triaging the above issue so that it is both covered in end-to-end tests, and the dates will have an offset. Thanks!